tests/test_features/test_importances.py

```diff
@@ -248,6 +249,24 @@ def test_multi_coefs(self):
 
         npt.assert_equal(visualizer.feature_importances_.ndim, 1)
 
+    @pytest.mark.xfail(
+        sys.platform == 'win32', reason="images not close on windows"
+    )
+    def test_multi_coefs_stacked(self):
+        """
+        Test stack plot with multidimensional coefficients
+        """
+        X_iris, y_iris = load_iris(True)
+        X_iris_pd = pd.DataFrame(X_iris, columns=['f1', 'f2', 'f3', 'f4'])
```

Comment:
Technically, pandas is not a dependency of Yellowbrick, even though it is a test dependency; still just to guard against confusion, we usually do `pytest.mark.skipif(pd is None)`; I will go ahead and make this change when I merge though - no worries!